roles/rhscl-postgresql-apb/tasks/main.yml

```diff
@@ -100,4 +100,5 @@
       DB_USER: "{{ postgresql_user }}"
       DB_PASSWORD: "{{ postgresql_password }}"
       DB_NAME: "{{ postgresql_database }}"
+      DB_ADMIN_PASSWORD: "{{ postgresql_admin_password }}"
```

Comment:
Just to clarify one thing:
The only reason we are still running `asb_encode_binding` on provision is to support 3.7 functionality right? Moving forward we would only be calling `asb_encode_binding` on the bind operation. True? Or am I mistaken?

Reply:
I think that module should is more of a `encode_credentials` then specifically binding.

We will still need this to retrieve the admin credentials during provision.